spring-cloud-gcp-samples/spring-cloud-gcp-config-sample/README.adoc

```diff
@@ -3,6 +3,9 @@
 This sample application demonstrates using the
 link:../../spring-cloud-gcp-starters/spring-cloud-gcp-starter-config[Spring Cloud GCP Config Client Starter] in your code.
 
+The same content is also available in
+https://codelabs.developers.google.com/codelabs/cloud-spring-runtime-config/index.html[codelab format].
```

Comment:
Can you please include the title of the codelab in the link please? Same goes for other links.

Reply:
@meltsufin Updated with more descriptive titles, PTAL?